src/main/java/yevhenii/lostfilmdemo/services/impl/ConvertServiceImpl.java

```diff
@@ -0,0 +1,44 @@
+package yevhenii.lostfilmdemo.services.impl;
+
+import org.springframework.stereotype.Service;
+import yevhenii.lostfilmdemo.entity.TVSeries;
+import yevhenii.lostfilmdemo.rss.FeedMessage;
+import yevhenii.lostfilmdemo.services.ConvertService;
+
+@Service
+public class ConvertServiceImpl implements ConvertService {
```

Comment:
instead of creating own interface you can implement existing one org.springframework.core.convert.converter.Converter